core/src/main/java/org/stellar/anchor/client/ClientConfig.java

```diff
@@ -1,7 +1,21 @@
 package org.stellar.anchor.client;
 
+import lombok.Builder;
+import lombok.Data;
+
 public interface ClientConfig {
   String getName();
 
   String getCallbackUrl();
+
+  CallbackUrls getCallbackUrls();
+
+  @Data
+  @Builder
+  class CallbackUrls {
+    private String sep6;
```

Comment:
> How does the handler uses the newly added `callback_urls`?

We don't use it right now. `develop` currently uses the old client config schema, when I merge `develop` into `feature/v3`, the implementation will be updated. I published this PR to reduce the implementation work needed for the merge.

> Can we also reflect this change in the` anchor_platform_default_values.yaml`?

The client configs no longer live in the anchor config. There is no change required to `anchor_platform_default_values`.

Reply:
When we add fields to the config, it helps to add comments to the config.

Reply:
>  I published this PR to reduce the implementation work needed for the merge.
Can you elaborate a little? Is there a reason that we cannot add implementation in the same PR?

Reply:
> When we add fields to the config, it helps to add comments to the config.

I just checked this there is a comment block about the client config. I'll add it.

Reply:
>I published this PR to reduce the implementation work needed for the merge.
> Can you elaborate a little? Is there a reason that we cannot add implementation in the same PR?

The existing implementation in `develop` uses the old client config format. The callback URLs are flat strings rather than nested in a `callbackUrls` object. To merge `develop` into `feature/v3`, I will need to move this implementation into the new client config format. This means we need to implement the changes in this PR. I think it's better to keep the merge PR light in additional implementation.

Reply:
We are planning to switch the clients config back to a flatter format. Maybe it would be a good idea to merge this PR after the refactoring?

Reply:
This shouldn't be majorly impacted by the refactoring and is currently blocking merging `develop` into `feature/v3`.